flink-table/flink-table-common/src/main/java/org/apache/flink/table/descriptors/DescriptorProperties.java

```diff
@@ -103,6 +103,18 @@ public void putProperties(DescriptorProperties otherProperties) {
 		}
 	}
 
+	/**
+	 * Adds a properties map by appending the given prefix to element keys.
+	 */
```

Comment:
Complete the javadoc as this is going to be a public method. Same for get.

Reply:
You mean `@param`? The style of existing javadoc in this class seems not favor it

Reply:
then it's okay.